website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,292 @@
+| support      | Function |    Yes    |                            Report a bug or create a support ticket with OpenBB. |
+| survey       | Function |     No     |                                    Take a short user survey to help us improve. |
+| update       | Function |     No     |   Attempt to update (**Only for Github cloned repository installations**) |
+| wiki         | Function |    Yes    |                                   Query the Wikipedia API for a term or phrase. |
+
+### Additional Global Commands
+
+The commands listed below are not displayed on a Terminal menu, but are active globally.
+
+| Function Key |                                       Description |
+| :----------- | ------------------------------------------------: |
+| cls          |                       Clears the Terminal screen. |
+| exit         |                               Quits the Terminal. |
+| help, h, ?   |                   Prints the current menu screen. |
+| quit, q, ..  |             Navigates back one menu towards Home. |
+| reset, r     | Resets the Terminal, opening to the current menu. |
+
+## Data
+
+Many functions will require obtaining (free or subscription) API keys from various data providers.  OpenBB provides methods for consuming these data feeds but has no control over the quality or quantity of data provided to an end-user.   None are required to get started using the Terminal.  See the list of data providers [here](https://docs.openbb.co/terminal/usage/guides/api-keys), along with instructions for entering the credentials into the OpenBB Terminal.  [Request a feature](https://openbb.co/request-a-feature) to let us know what we are missing!
+
+### Default Data Sources
+
+The default data source for each function (where multiple sources are available) can be defined within the [`/sources` menu](https://docs.openbb.co/terminal/usage/guides/changing-sources).  The available sources for each function are displayed on the right of the menu, and they can be distinguished by the square brackets and distinct font color group.  Unless a preference for a particular function is defined, the command will prioritize in the order they are displayed, from left-to-right, on the Terminal screen.  To override a preference or default source, select one of the other choices by attaching the, `--source`, argument to the command syntax.  The available sources for the feature will be populated by auto complete when the `space bar` is pressed after typing `--source`.  This information is also printed with the `--help` dialogue of a command.
+
+```console
+/stocks/load AAPL/fa/earnings --source YahooFinance
 ```
 
-![Candlestick Chart Amazon](https://user-images.githubusercontent.com/46355364/223211728-1317abea-36da-461c-bc3b-140ed7973173.svg)
+![Selecting a new Data Source](https://user-images.githubusercontent.com/85772166/233719604-9f28b80d-4c5f-41f5-8cfe-71d5aaa6da99.png)
```

Comment:
I think this image in unnecessary.    I was envisioning an example of how to set, so something like

` set --cmd stocks/fa/income --source FinancialModelingPrep`

But maybe a close zoom in of the function showing --source does also make sense to show.

Reply:
Ah, I was making an example of a preference being overridden and not set with the `/sources` menu.

Reply:
Updated to just be a zoomed shot of the source choices.